File: bd1903_javaSE/src/day8/HomeWork.java
package day8;

public class HomeWork {
    public static void main(String[] args) {
        B b = new B();
        System.out.println(b.a);
        b.b();

    }
}
class A{
    /*private*/ int a;
    /*private*/ void b(){}
}
class B extends A{

}

File: bd1903_spark/bd1903_scala/src/main/scala/com/briup/bigdata/day02/HomeWork.scala
package com.briup.bigdata.day02

object HomeWork {
  def main(args: Array[String]): Unit = {
    getNums.foreach(println)
  }
  def getNums :List[Int]={
    var list:List[Int] = List.apply();
    for(num:Int <- 100 to 999){
      var g = num%10
      var s = (num/10)%10
      var b = (num/100)%10
      if(((g*g*g)+(s*s*s)+(b*b*b))==num){
        list = num::list
      }
    }
    list
  }
}
